packages/contracts-bridge/contracts/test/utils/BridgeTest.sol

```diff
@@ -96,21 +103,25 @@ contract BridgeTest is Test {
         bridgeRouter.enrollRemoteRouter(remoteDomain, remoteBridgeRouter);
     }
 
-    function createRemoteToken() public {
-        remoteTokenAddress = tokenRegistry.deployToken(
-            remoteDomain,
-            addressToBytes32(vm.addr(999999999))
+    /// @notice Create the representation of a remote token
+    /// @param newRemoteDomain The remote domain
+    /// @param remoteAddress The id of the remote token, which also happens
+    /// to be the address of the token on that domain
+    function createRemoteToken(uint32 newRemoteDomain, bytes32 remoteAddress)
```

Comment:
this might be a bit late, but this is creating a representation not a remote token right? I think that's why I'm having trouble reading al ot of the tests. as they're written as "create remote"